MN/Algoritmi/LU/Cholesky/Cholesky.m

```diff
@@ -0,0 +1,65 @@
+% calculates Cholesky factorization, A = L * L'
+% A must be symmetrical and positive-definite
+function [L U x success] = Cholesky (A, b) 
```

Comment:
nu mai ai nevoie de success daca faci x = NaN in caz ca nu e success, deci renunta la el.